3p/beopinion.js

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
the ad part should belong to ads/ folder

Reply:
Thanks a lot for the review @lannka ! I'm working on figuring out which part I could put in `ads` but our solution is mainly 3p and not ad. The ad part is just a mode in which the extension can be used.

Reply:
with the current implementation, user would be able to use your service in AMP in 2 ways:

```
<amp-ad type="beopinion">
``` 
or
```
<amp-beopinion>
```

do you want both? what's the difference?

Reply:
Yes, we actually want to use both. 
- A publisher would use an `<amp-beopinion>` tag to create their own quiz or poll for instance. 
- But if they agree to let brands use a space on their page to ask their own questions, they'll have to do it by adding an `<amp-ad type="beopinion">` tag.

Reply:
So to do <amp-ad type="beopinion">, you will need move 3p/beopinion.js to ads/beopinion.js. the other code remain the same

Reply:
Hi @lannka,
I hope you're doing fine.
Unfortunately, when I move `3p/beopinion.js` to `ads/beopinion.js`, I am unable to use an `amp-beopinion` tag.